packages/zero-tag-processor/src/preprocessor.ts

```diff
@@ -0,0 +1,29 @@
+import type { Element } from 'stylis';
+import { serialize, compile, stringify, middleware } from 'stylis';
+
+function globalSelector(element: Element) {
+  switch (element.type) {
+    case 'rule':
+      element.props = (element.props as string[]).map((value: any) => {
+        if (value.match(/(:where|:is)\(/)) {
```

Comment:
@siriwatknp can you check this please?

Reply:
It's copied over from [here](https://github.com/mui/material-ui/blob/master/docs/src/createEmotionCache.ts#L5).

Reply:
This is a workaround since emotion has an issue with `:where` and `:is` selector.

Do we need it here? I am not quite sure.

Reply:
It's added by default so that generated css doesn't have the `*` selector in it.